src/yang/ietf-bgp-capabilities.yang

```diff
@@ -0,0 +1,339 @@
+submodule ietf-bgp-capabilities {
+  yang-version 1.1;
+  belongs-to ietf-bgp {
+    prefix bgp;
+  }
+
+  import iana-bgp-types {
+    prefix bt;
+    reference
+      "RFC XXXX: BGP Model for Service Provider Network.";
+  }
+  import iana-routing-types {
+    prefix iana-rt-types;
+  }
+  import ietf-inet-types {
+    prefix inet;
+    reference
+      "RFC 6991: Common YANG Data Types.";
+  }
+
+  organization
+    "IETF IDR Working Group";
+  contact
+    "WG Web:   <http://tools.ietf.org/wg/idr>
+     WG List:  <idr@ietf.org>
+
+     Authors: Mahesh Jethanandani (mjethanandani at gmail.com),
+              Keyur Patel (keyur at arrcus.com),
+              Susan Hares (shares at ndzh.com,
+              Jeffrey Haas (jhaas at juniper.net).";
+
+  description
+    "This sub-module contains groupings that used for BGP 
+     Capabilities within the BGP module.
+
+     Copyright (c) 2023 IETF Trust and the persons identified as
+     authors of the code. All rights reserved.
+
+     Redistribution and use in source and binary forms, with or
+     without modification, is permitted pursuant to, and subject to
+     the license terms contained in, the Revised BSD License set
+     forth in Section 4.c of the IETF Trust's Legal Provisions
+     Relating to IETF Documents
+     (https://trustee.ietf.org/license-info).
+
+     This version of this YANG module is part of RFC XXXX
+     (https://www.rfc-editor.org/info/rfcXXXX); see the RFC itself
+     for full legal notices.
+
+     The key words 'MUST', 'MUST NOT', 'REQUIRED', 'SHALL', 'SHALL
+     NOT', 'SHOULD', 'SHOULD NOT', 'RECOMMENDED', 'NOT RECOMMENDED',
+     'MAY', and 'OPTIONAL' in this document are to be interpreted as
+     described in BCP 14 (RFC 2119) (RFC 8174) when, and only when,
+     they appear in all capitals, as shown here.";
+
+  revision YYYY-MM-DD {
+    description
+      "Initial Version";
+    reference
+      "RFC XXXX, BGP Model for Service Provider Network.";
+  }
+
+  grouping bgp-capabilities-common {
+    description
+      "BGP Capabilities that are used commonly in by the
+       advertised-capabilities and received-capabilities lists in the
+       bgp-capabilities grouping.";
+
+    leaf code {
+      type uint8;
+      description
+        "BGP Capability Code";
+      reference
+        "RFC 5492: Capabilities Advertisement with BGP-4, Section
+         4."; }
+    leaf index {
+      type uint8;
+      description
+        "Multiple BGP Capabilities with a given Capability Code may
+         be advertised in a BGP OPEN message.  This index permits
+         multiple such Capabilities to be represented in the YANG
+         model.  It is RECOMMENDED that this index start at one (1)
+         and increase by one for each additional instance of the
+         Capability.";
+      reference
+        "RFC 5492: Capabilities Advertisement with BGP-4, Section
+         4."; }
```

Comment:
Again, move the closing brace on a new line.

Reply:
Fixed in next patch.